php/example_code/glue/setup_scenario_getting_started.yaml

```diff
@@ -0,0 +1,164 @@
+      aws:cdk:path: doc-example-glue-scenario-stack/CDKMetadata/Default
+    Condition: CDKMetadataAvailable
+Outputs:
+  BucketName:
+    Value:
+      Ref: docexampleglue6E2F12E5
+  RoleName:
+    Value:
+      Ref: AWSGlueServiceRoleDocExampleF579DC66
+Conditions:
+  CDKMetadataAvailable:
+    Fn::Or:
+      - Fn::Or:
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - af-south-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - ap-east-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - ap-northeast-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - ap-northeast-2
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - ap-south-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - ap-southeast-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - ap-southeast-2
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - ca-central-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - cn-north-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - cn-northwest-1
+      - Fn::Or:
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - eu-central-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - eu-north-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - eu-south-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - eu-west-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - eu-west-2
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - eu-west-3
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - me-south-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - sa-east-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - us-east-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - us-east-2
+      - Fn::Or:
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - us-west-1
+          - Fn::Equals:
+              - Ref: AWS::Region
+              - us-west-2
+#Parameters:
+#  BootstrapVersion:
+#    Type: AWS::SSM::Parameter::Value<String>
+#    Default: /cdk-bootstrap/hnb659fds/version
+#    Description: Version of the CDK Bootstrap resources in this environment, automatically retrieved from SSM Parameter Store. [cdk:skip]
+#Rules:
+#  CheckBootstrapVersion:
+#    Assertions:
+#      - Assert:
+#          Fn::Not:
+#            - Fn::Contains:
+#                - - "1"
+#                  - "2"
+#                  - "3"
+#                  - "4"
+#                  - "5"
+#                - Ref: BootstrapVersion
+#        AssertDescription: CDK bootstrap stack version 6 required. Please run 'cdk bootstrap' with a recent version of the CDK CLI.
```

Comment:
Please run --> Run
Does "CDK CLI" refer to the "AWS CDK Toolkit"? https://docs.aws.amazon.com/cdk/v2/guide/cli.html

Reply:
Yes, that's what it's referring to. But I realized I had commented out that section as, so I removed everything past line 145 and this is no longer strictly relevant.